bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -506,5 +507,347 @@
 		<description>Away Mode (ON or OFF)</description>
 	</channel-type>
 
+	<channel-type id="zone_mode">
```

Comment:
Please update the default translations (use the maven command for an automatic update).

Reply:
I have never used it before and I am getting error ([ERROR] No plugin found for prefix 'i18n' in the current project) when I run 
```mvn i18n:generate-default-translations```
Java 11, maven 3.6.0
Any idea what I am doing wrong? \
Thanks

Reply:
Please try : mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

Reply:
solved with the 3.2.0 version